tests/sentry/seer/similarity/test_grouping_records.py

```diff
@@ -142,13 +144,14 @@ def test_post_bulk_grouping_records_use_reranking(
         json.dumps(expected_return_value).encode("utf-8"), status=200
     )
 
-    CREATE_GROUPING_RECORDS_REQUEST_PARAMS.update({"use_reranking": True})
```

Comment:
this was the bug right here -- mutating a shared global used by all the other tests!